submissions/SerhiiYakovenko/html-css-popup/index.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>CyberPunk</title>
+    <link href="./css/style.css" rel="stylesheet" />
+    <link rel="icon" href="./img/favicon.png" type="image/png" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Orbitron&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+
+  <body>
+    <nav class="nav">
+      <ul class="nav__menu menu">
+        <li class="nav-item headquarter">
+          <a
```

Comment:
I'd recommend you adding a border styles `1.5px solid transparent` for the default state of the links with the classes `menu-item` and `popup-item` in order to prevent content shifting because on each hover or focus state I see the content shifting.